File: MTWDM iOS Xamarin/App7/App7/App7/ViewController.cs
using System;
using Foundation;
using UIKit;

namespace App7
{
    public partial class ViewController : UIViewController
    {
        protected ViewController(IntPtr handle) : base(handle)
        {
            // Note: this .ctor should not contain any initialization logic.
        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
            // Perform any additional setup after loading the view, typically from a nib.
        }

        public override void PrepareForSegue(UIStoryboardSegue segue, NSObject sender)
        {
            base.PrepareForSegue(segue, sender);

            var viewController = segue.DestinationViewController as SegundaViewController;
            viewController.MsgEnviado = EditorTexto.Text;

        }

        public override void DidReceiveMemoryWarning()
        {
            base.DidReceiveMemoryWarning();
            // Release any cached data, images, etc that aren't in use.
        }
    }
}

File: MTWDM iOS Swift /app7/app7/ViewController.swift
//
//  ViewController.swift
//  app7
//
//  Created by Felipe Hernandez on 10/7/18.
//  Copyright © 2018 Felipe Hernandez. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var editorTexto: UITextView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        editorTexto.becomeFirstResponder()
    }

    //PrepareForSegue sirva para pasar datos de una UIViewController a otra por medio de los segue´s
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        
        MensajePropGlobal = editorTexto.text
    }

}

